### PR TITLE
Fixes #7203: fusionAgent.cf for ARM

### PR DESCRIPTION
**Pull request by frbayart**
- remove hidden charaters
- avoid dmidecode usage on armv6l and armv7l but read Serial line in /proc/cpuinfo

### CODE REVIEW

File: techniques/system/inventory/1.0/fusionAgent.st [hunk 1]
```diff
@@ -58,16 +58,16 @@ bundle agent doInventory
 
   vars:
 
-    # If curl is available, use it
+    # If curl is available, use it
```
Comment (peckpeck) [inline techniques/system/inventory/1.0/fusionAgent.st]: thanks for these :)

File: initial-promises/node-server/inventory/1.0/fusionAgent.cf [hunk 2]
```diff
@@ -421,6 +427,9 @@ bundle agent addInformationsToInventory
     xen::
       "xenrudderuuid" expression => regextract("/vm/(.*)", "${VMRUDDERUUID}", "vmarray");
 
+    linux.(armv6l|armv7l)::
+      "armv6lorv7lrudderuuid" expression => regextract("Serial[\s]+:[\s](.*)\N", "$(ARMCPUINFO)", "serialarray");
```
Comment (peckpeck) [inline initial-promises/node-server/inventory/1.0/fusionAgent.cf]: We usually use ${} instead of $()

**Comment by peckpeck** [comment]
Thank you, that's good to see new contributor and see a quality patch.
Would you be kind enough to correct the $() for the correctness nazi I am ?
Otherwise I will do it during merge.
If you are using rudder-dev, it's a matter of: edit file; git add file; rudder-dev amend

**Comment by peckpeck** [comment]
We'd like you to sign one of our CLA: 
- for individuals -> https://secure.echosign.com/public/hostedForm?formid=ECPMWD7B3BU3H
- for company -> https://secure.echosign.com/public/hostedForm?formid=4YD3AXT4S76F5J

You can find in our wiki the reason we ask for it http://www.rudder-project.org/foswiki/Development/HowToContribute#Why_must_I_sign_a_CLA_63_Do_you_want_to_make_money_with_my_code_63

Thanks again for contributing.

**Comment by frbayart** [comment]
I have replaced $() to ${}

**Comment by peckpeck** [comment]
Hello @frbayart.
Tince this PR cannot be merged anymore, I took your commits and put them in a new PR https://github.com/Normation/rudder-techniques/pull/970
Thank you for your time

**Pull request closed without merging**